src/built_in/ft_cd.c

```diff
@@ -0,0 +1,44 @@
+#include "../../include/minishell.h"
+
+int go_to_home_directory(t_list *env_list, t_token *token_list, char **path)
+{
+    if (token_list == NULL || token_list->value[0] == '~'
+        || (token_list->value[0] == '~' && token_list->value[1] == '/'))
+    {
+        *path = get_env_by_key(env_list, "HOME")->value;
+        return (1);
+    }
+    return (0);
+}
+
+void    go_to_path_directory(t_token *token_list, char **path)
+{
+    *path = token_list->value;
+}
+
+void    update_env_pwd(t_list **env_list, t_env **change_pwd)
+{
+    *change_pwd = get_env_by_key(*env_list, "PWD");
+    free((*change_pwd)->value);
+    (*change_pwd)->value = NULL;
+    (*change_pwd)->value = getcwd(NULL, 0);
+}
+
+void  ft_cd(t_node *minishell)
```

Comment:
```suggestion
void  ft_cd(t_minishell *minishell)
```

구조체 이름  t_node 에서 t_minishell 로 바뀌었습니다.  node 라는 구조체는 너무 범용적이어서

Reply:
네. 이건 머지할 때 수정할게요.